backend/app/assets/javascripts/spree/backend/navigation.coffee

```diff
@@ -4,7 +4,18 @@ navHeight = ->
 checkSideBarFit = ->
   $('.admin-nav').toggleClass('fits', navHeight() < $(window).height())
 
-$ ->
-  $(".admin-nav-sticky, .admin-nav").stick_in_parent()
+checkSticky = ->
+  stickyElements = $('.admin-nav-sticky, .admin-nav')
+
+  if $(window).width() < 768
+    stickyElements.trigger 'sticky_kit:detach'
+  else
+    stickyElements.stick_in_parent(spacer: false)
```

Comment:
@tvdeyen should be fixed with https://github.com/solidusio/solidus/pull/1718/commits/8e9a39f3ad78bda18324b46ed4e73462556116f6, what do you think?